index.html

```diff
@@ -618,6 +618,93 @@ <h4>
         </section>
       </section>
     </section>
+    <section id="automation" class="informative">
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document define [[WEBDRIVER2]]
+        <a href="https://w3c.github.io/webdriver/#dfn-extension-commands">
+        extension command</a> that allow controlling device posture like in real
+        device.
+      </p>
+      <h3>
+        Extension Commands
+      </h3>
+      <h4>
+        Set device posture
+      </h4>
+      <table class="simple">
+        <tr>
+          <th>
+            HTTP Method
+          </th>
+          <th>
+            <a href="https://w3c.github.io/webdriver/#dfn-extension-command-uri-template">
+            URI Template</a>
+          </th>
+        </tr>
+        <tr>
+          <td>
+            POST
+          </td>
+          <td>
+            window/deviceposture
+          </td>
+        </tr>
+      </table>
+      <p>
+        This <a href="https://w3c.github.io/webdriver/#dfn-extension-commands">
+        extension command</a> changes device posture to certain
```

Comment:
```suggestion
        extension command</a> changes device posture to a specific
```

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/e8b4c90bfb60199245dae6f40807bf2e6fabd249